python_modules/dagster-graphql/dagster_graphql/schema/roots/query.py

```diff
@@ -549,14 +549,14 @@ def resolve_partitionSetsOrError(self, graphene_info, **kwargs):
         return get_partition_sets_or_error(
             graphene_info,
             RepositorySelector.from_graphql_input(kwargs.get("repositorySelector")),
-            kwargs.get("pipelineName"),
+            kwargs["pipelineName"],
         )
 
     def resolve_partitionSetOrError(self, graphene_info, **kwargs):
         return get_partition_set(
             graphene_info,
             RepositorySelector.from_graphql_input(kwargs.get("repositorySelector")),
-            kwargs.get("partitionSetName"),
+            kwargs["partitionSetName"],
```

Comment:
Previous behavior is that `check.str_param(partition_set_name, "partition_set_name")` throws-- not a peaceful return of `GraphenePartitionSetNotFoundError`. But I'll revert this change since that check error is handled by a `@capture_error`.